### ISSUE TITLE
Photoswipe dismissed on cloise when opened with picture link

### ISSUE THREAD

**Issue by DanielMuller** [DanielMuller/koken-plugin-photoswipe#25]
Reported by @laklare in #22. Opening as new issue for easier tracking.

I'm getting some new odd behavior in Regale 2, but not Madison 2 or Axis 2. When I paste a link to a photo, I get the photoswipe lightbox with the correct image displayed just fine. However, when I click the close button and return to grid view, photoswipe won't open again and the standard koken lightbox comes up when I click on an image. This behavior fixes itself if I scroll down far enough to trigger the lazy-loading of additional images. After that, photoswipe works again. This only happens for me in Regale 2...for some reason photoswipe isn't being initialized at first when there is a pid and gid in the URL.

**Comment by DanielMuller** [comment]
I am unable to reproduce the issue. Did you delete system caches before?
Any error displayed in the browser console?
I didn't push a minified version of pswp.js, perhaps that's the issue.

**Comment by DanielMuller** [comment]
I just pushed [v1.7.1](/DanielMuller/koken-plugin-photoswipe/releases/tag/v1.7.1). Is this release solving your problem?

**Comment by laklare** [comment]
I did make my own minified version of pswp.js to test the changes you made prior to v1.7.1. I just tried v1.7.1 and had the same issue with Regale 2 on my site. I have been clearing system caches after each change to the plugin and there is no error in the Chrome console. I tried other browsers as well. 

I just set up a new koken site and applied Regale 2 and had no issue. Photoswipe is opening fine every time after pasting a URL. So there must be something on my site...disabled my custom CSS, but perhaps a plugin is messing up photoswipe in Regale 2. I'll reply again if I figure it out, but the main problem I had with not being able to link directly to photos has been fixed. Thanks
